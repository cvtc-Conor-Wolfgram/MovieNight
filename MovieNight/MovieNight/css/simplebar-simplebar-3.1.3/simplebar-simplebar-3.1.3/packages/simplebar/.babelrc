{
  "presets": [
    [
      "@babel/preset-env",
      {
        "modules": false,
        "useBuiltIns": "usage"
      }
    ]
  ],
  "plugins": [
    "transform-class-properties"
  ],
  "env": {
    "test": {
      "presets": [
        [
          "@babel/preset-env"
        ]
      ]
    }
  }
}