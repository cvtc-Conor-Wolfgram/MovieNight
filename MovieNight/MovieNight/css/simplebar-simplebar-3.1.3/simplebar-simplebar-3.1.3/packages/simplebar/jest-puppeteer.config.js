module.exports = {
    launch: {
        headless: process.env.CI === 'true',
    },
    server: {
        command: 'webpack-dev-server --port 8091 --mode=development',
        port: 8091
    }
}