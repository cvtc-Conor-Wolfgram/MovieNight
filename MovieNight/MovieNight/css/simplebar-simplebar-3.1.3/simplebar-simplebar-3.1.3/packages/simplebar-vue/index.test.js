import { shallowMount } from '@vue/test-utils'
import simplebar from './index.vue'

describe('simplebar', () => {

  it('renders without crashing', () => {
    const wrapper = shallowMount(simplebar);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders with options', () => {
    const wrapper = shallowMount(simplebar, {
      propsData: { 'data-simplebar-auto-hide': 'false' }
    });
    expect(wrapper).toMatchSnapshot();
  });

});
