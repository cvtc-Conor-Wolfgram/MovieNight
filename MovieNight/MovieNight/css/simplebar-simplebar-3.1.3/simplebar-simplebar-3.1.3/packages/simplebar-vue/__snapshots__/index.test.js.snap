// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simplebar renders with options 1`] = `
<div data-simplebar="" data-simplebar-auto-hide="false">
  <div class="simplebar-wrapper">
    <div class="simplebar-height-auto-observer-wrapper">
      <div class="simplebar-height-auto-observer"></div>
    </div>
    <div class="simplebar-mask">
      <div class="simplebar-offset">
        <div class="simplebar-content"></div>
      </div>
    </div>
    <div class="simplebar-placeholder"></div>
  </div>
  <div class="simplebar-track simplebar-horizontal">
    <div class="simplebar-scrollbar"></div>
  </div>
  <div class="simplebar-track simplebar-vertical">
    <div class="simplebar-scrollbar"></div>
  </div>
</div>
`;

exports[`simplebar renders without crashing 1`] = `
<div data-simplebar="">
  <div class="simplebar-wrapper">
    <div class="simplebar-height-auto-observer-wrapper">
      <div class="simplebar-height-auto-observer"></div>
    </div>
    <div class="simplebar-mask">
      <div class="simplebar-offset">
        <div class="simplebar-content"></div>
      </div>
    </div>
    <div class="simplebar-placeholder"></div>
  </div>
  <div class="simplebar-track simplebar-horizontal">
    <div class="simplebar-scrollbar"></div>
  </div>
  <div class="simplebar-track simplebar-vertical">
    <div class="simplebar-scrollbar"></div>
  </div>
</div>
`;
